app/lib/assessment_factory.rb

```diff
@@ -231,6 +231,7 @@ def work_history_section
           FailureReasons::SCHOOL_DETAILS_CANNOT_BE_VERIFIED,
           FailureReasons::UNRECOGNISED_REFERENCES,
           FailureReasons::WORK_HISTORY_DURATION,
+          FailureReasons::SATISFACTORY_EVIDENCE_WORK_HISTORY,
```

Comment:
Is it correct that we're wanting to add this failure reason to the new regulations? It doesn't seem like a strictly related change.